### PR TITLE
chore(Breadcrumbs,DateInput,ListBox,NotificationCard,SideNavigation,T…

### PR DESCRIPTION
**Pull request by jamiek-acl**
…abs,Uploader): ISSUE-873 Resolve console errors/warnings when run tests

No more warnings and errors cluttering up the console. Just green, baby!

https://github.com/acl-services/paprika/issues/873

Screen Shot 2021-04-29 at 2 24 27 PM
Mr _Clean_logo

### CODE REVIEW

File: packages/DateInput/test/DateInput.spec.js [hunk 1]
```diff
@@ -48,6 +48,8 @@ describe("DateInput", () => {
   });
 
   it("should show error state if it cannot parse the typing string", () => {
+    jest.spyOn(console, "warn").mockImplementation(() => {}); // suppress a momentjs warning
+
```
Comment (allison-c) [inline packages/DateInput/test/DateInput.spec.js:52]: will this mock be cleared after all tests run?

**Comment by nahumzs** [comment]
this is great!!

File: packages/DateInput/test/DateInput.spec.js (hunk 1 above)
Comment (jamiek-acl) [reply packages/DateInput/test/DateInput.spec.js:52]: oh allison, so smart!

Comment (jamiek-acl) [reply packages/DateInput/test/DateInput.spec.js]: Eagle-eye
download

Comment (allison-c) [reply packages/DateInput/test/DateInput.spec.js]: 😹

**Review by allison-c — APPROVED** [review]

**Pull request merged**